example/stochastic_subgrid/stochastic_subgrid_driver.cpp

```diff
@@ -134,15 +134,15 @@ TaskCollection StochasticSubgridDriver::MakeTaskCollection(BlockList_t &blocks,
 
       const auto any = parthenon::BoundaryType::any;
 
-      tl.AddTask(none, parthenon::cell_centered_bvars::StartReceiveBoundBufs<any>, mc1);
-      tl.AddTask(none, parthenon::cell_centered_bvars::StartReceiveFluxCorrections, mc0);
+      tl.AddTask(none, parthenon::var_boundary_comm::StartReceiveBoundBufs<any>, mc1);
```

Comment:
Is the `var_boundary_comm` namespace necessary? Was this just intended to separate cell-centered vars from others, and will this distinction be necessary to the current plan for face/edge fields?

Reply:
I think I agree with you that the `var_boundary_comm` namespace is unnecessary and was probably put there to distinguish between different types of fields. I guess the only place it could conflict is for swarms (?), since the plan is to unify communication of all types of grid variables. I will leave this up to see if others have thoughts.

Reply:
If we don't remove the name space, would `var_comms` be appropriate? It's slightly less verbose and more in line with the `comms` directory

Reply:
Hmm... I think we don't need to preserve this namespace just to deconflict with swarms. Although for swarm packing we will probably want to move to a model like this, e.g. `StartReceiveSomething(std::shared_ptr<MeshData>)` , the communicated swarms aren't really "boundary buffers" (the particles being communicated are literally already on the meshblock they are being moved to) it would be natural for me to just use different names like `StartReceiveSwarmBufs` etc.

Reply:
Ok, I am going to remove the namespace entirely.